src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,144 @@
+
+/**
+* @method init
+* @param {jQuery Event} event Event that triggered the function call
+*/
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm,
+			settings;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+		true,
+		defaults, {},
+		window[ componentName ],
+		wb.getData( $elm, componentName )
+	);
+			i18nDict = i18nDict[ $( "html" ).attr( "lang" ) || "en" ];
+			i18nDict = {
+				warning: i18nDict[ "exitscript-warning" ],
+				exitMsg: i18nDict[ "exitscript-exitMsg" ],
+				yesMsg: i18nDict[ "exitscript-yesMsg" ],
+				cancelMsg: i18nDict[ "exitscript-cancelMsg" ]
+			};
+			$elm.trigger( "setModaldialog", settings );
+			var exitParam = "?" + encodeURIComponent( "targetUrl=" + String( this.href ) );
+
+			if ( settings.targetBlank ) {
+				exitParam = exitParam + encodeURIComponent( "&&targetBlank=true" );
+			}
+
+			// Set default propertie
+
+			if ( settings.displayModal !== false ) {
+				if ( !settings.targetBlank ) {
+					settings.targetBlank = "";
+				} else {
+					settings.targetBlank = "target='_blank'";
+				}
+				if ( !settings.exitMsg ) {
+					settings.exitMsg = i18nDict.exitMsg;
+				}
+				if ( !settings.yesMsg ) {
+					settings.yesMsg = i18nDict.yesMsg;
+				}
+				if ( !settings.cancelMsg ) {
+					settings.cancelMsg = i18nDict.cancelMsg;
+				}
+				var tpl = document.createElement( "div" );
+				tpl.id = "tm";
+				tpl.innerHTML = "<section id='sExitModal' class='mfp-hide modal-dialog modal-content overlay-def'>" +
+			"<header class='modal-header'><h2 class='modal-title'>" + i18nDict.warning + "</h2></header>" +
+			"<div class='modal-body'>" +
+			"<p>" + settings.exitMsg + "</p>" +
+			"<ul class='list-inline text-center'>" +
+			"<li><button class='btn btn-primary popup-modal-dismiss pull-left' id='eCancel'>" + settings.cancelMsg + "</button></li>" +
+			"<li><a class='btn btn-default pull-right popup-modal-close' " + settings.targetBlank + " href='" + settings.exitURL + exitParam + "' type='button'>" + settings.yesMsg + "</a></li>" +
+			"</ul></div></section>";
+				var moDal = document.createDocumentFragment();
+				moDal.appendChild( tpl );
+				document.body.appendChild( moDal );
+
+		//Unwrap div id="tm"
+				var wrapper = document.querySelector( "#tm" );
+				wrapper.outerHTML = wrapper.innerHTML;
+			}
+			wb.ready( $elm, componentName );
+		}
+
+	};
+
+// Add your plugin event handler
+$document.on( "click", selector, function( ev ) {
+	ev.preventDefault();
+
+	var settings = $.extend(
+	defaults, {}
+);
+
+	var exitParam = "?" + encodeURIComponent( "targetUrl=" + String( this.href ) );
+
+	if ( settings.targetBlank ) {
+		exitParam = exitParam + encodeURIComponent( "&&targetBlank=true" );
+	}
+
+	if ( settings.displayModal !== false ) {
+		$document.trigger( "open.wb-lbx", [
+			[ {
+				src: "#sExitModal",
+				type: "inline"
+			} ],
+			true
+		] );
+
+	} else {
+
+		if ( settings.displayModal === false && settings.targetBlank !== true ) {
```

Comment:
I think for this case, the only thing the script will add value is to append query parameter to the href attribute. If the plugin require the author to set it on every single link, then will be preferable to show them to use the attribute `target='_blank'`